website/docs/patterns/overview.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Patterns
+status: comingSoon
+---
+
+## What is a pattern?
+
+A pattern is a combination of components that work together to accomplish a function or render a layout. They are often used to solve the same problem consistently. Common examples of patterns are forms, navigation, button sets, etc.
+
+## HDS pattern support
+
+We don't currently support patterns, but we have a few on our [roadmap](URL). We're also planning to introduce guidelines and best practices around assembling components in a consistent and scalable manner.
+
+!!! Info
+
+**Note:**
+
+There are a small number of HDS components that showcase a pattern-like level of complexity ([Modal](/components/modal), [Table](/components/table), [Pagination](/components/pagination)) but are considered components and published as such.
+!!!
+
+If you find yourself building the same pattern often, [submit a new pattern request](https://docs.google.com/forms/d/e/1FAIpQLScpMXgrUTVT5fYriu4Pp48r4Nl_eCPluVnJLg0Yg3NXsRWvIA/viewform). 
+
+Don't hesitate to contact the design system team for best practices and suggestions when building and defining local components and patterns.
```

Comment:
@jorytindall if you want to link to, now there is a `support` page/route you can use: https://hds-website-hashicorp.vercel.app/support/
just use this `<LinkTo @route="support">...</LinkTo>` (`<LinkTo>` is like `<a>` but for links within the app, so it doesn't trigger a page reload)